api/src/pcapi/routes/backoffice_v3/offerers.py

```diff
@@ -526,15 +526,19 @@ def list_offerers_attachments_to_validate() -> utils.BackofficeResponse:
         date_utils.date_to_localized_datetime(form.to_date.data, datetime.datetime.max.time()),
     )
 
-    sorted_users_offerers: BaseQuery = users_offerers.order_by(offerers_models.UserOfferer.id.desc())
+    sorted_users_offerers: BaseQuery = users_offerers.order_by(
+        getattr(getattr(offerers_models.UserOfferer, form.sort.data), form.order.data)()
+    )
 
     paginated_users_offerers = sorted_users_offerers.paginate(
         page=int(form.data["page"]),
         per_page=int(form.data["per_page"]),
     )
 
-    next_page = partial(url_for, ".list_offerers_attachments_to_validate", **form.raw_data)
-    next_pages_urls = search_utils.pagination_links(next_page, int(form.data["page"]), paginated_users_offerers.pages)
+    form_url = partial(url_for, ".list_offerers_attachments_to_validate", **form.raw_data)
+    next_pages_urls = search_utils.pagination_links(form_url, int(form.data["page"]), paginated_users_offerers.pages)
+
+    date_created_sort_url = form_url(sort="dateCreated", order="desc" if form.order.data == "asc" else "asc")
```

Comment:
Est-ce que ça ne devrait pas plutôt être 
```suggestion
    date_created_sort_url = form_url(sort="dateCreated", order="desc" if form.order.sort == "dateCreated" and form.order.data == "asc" else "asc")
```
parce que si on trie par autre chose, ça doit être par défaut en `"asc"` quand on clique sur ce lien, non ?

Reply:
Bien vue !

Reply:
Ça me fait remarquer que dans le template HTML, il y a le même soucis.